docs/docs/how-to/installing-shadcn.md

```diff
@@ -0,0 +1,347 @@
+ 
+    --secondary: 210 40% 96.1%;
+    --secondary-foreground: 222.2 47.4% 11.2%;
+ 
+    --accent: 210 40% 96.1%;
+    --accent-foreground: 222.2 47.4% 11.2%;
+ 
+    --destructive: 0 100% 50%;
+    --destructive-foreground: 210 40% 98%;
+ 
+    --ring: 215 20.2% 65.1%;
+ 
+    --radius: 0.5rem;
+  }
+ 
+  .dark {
+    --background: 224 71% 4%;
+    --foreground: 213 31% 91%;
+ 
+    --muted: 223 47% 11%;
+    --muted-foreground: 215.4 16.3% 56.9%;
+ 
+    --accent: 216 34% 17%;
+    --accent-foreground: 210 40% 98%;
+ 
+    --popover: 224 71% 4%;
+    --popover-foreground: 215 20.2% 65.1%;
+ 
+    --border: 216 34% 17%;
+    --input: 216 34% 17%;
+ 
+    --card: 224 71% 4%;
+    --card-foreground: 213 31% 91%;
+ 
+    --primary: 210 40% 98%;
+    --primary-foreground: 222.2 47.4% 1.2%;
+ 
+    --secondary: 222.2 47.4% 11.2%;
+    --secondary-foreground: 210 40% 98%;
+ 
+    --destructive: 0 63% 31%;
+    --destructive-foreground: 210 40% 98%;
+ 
+    --ring: 216 34% 17%;
+ 
+    --radius: 0.5rem;
+  }
+}
+ 
+@layer base {
+  * {
+    @apply border-border;
+  }
+  body {
+    @apply bg-background text-foreground;
+    font-feature-settings: "rlig" 1, "calt" 1;
+  }
+}
+```
+
+### 7. Add a cn helper
+
+Shad uses a `cn` helper to make it easier to conditionally add Tailwind CSS classes.
+
+Within the `web/src` directory, create a folder called `lib`, with a file inside called `utils.ts`
+
+Add the following code:
+```ts
+// web/src/lib/utils.ts
+import { clsx, type ClassValue } from 'clsx'
+import { twMerge } from 'tailwind-merge'
+
+export function cn(...inputs: ClassValue[]) {
+  return twMerge(clsx(inputs))
+}
+```
+
+### 8. That's it! ... Kind of. If you want to use ShadCN's command line interface (CLI) keep reading.
+
+If you look at [ShadCN's documentation for their CLI tool](https://ui.shadcn.com/docs/cli), you're supposed to run 
+
+```sh
+npx shadcn-ui@latest init
+```
+
+The problem is that this command installs the dependencies for ShadCN, adds the `cn` util, configures `tailwind.config.js`, and CSS variables for the project, and we already did all those steps! In fact, if you try running that command anyway, it will tell you that Tailwind does not exist and you need to install that first.
+
+One thing that the `init` command does, that we do need to add is a `components.json` file. This file tells ShadCN CLI where everything is located within our project.
+
+Within our `web` directory, create a `components.json` file. The contents for this file is [documented here](https://ui.shadcn.com/docs/components-json). Inside add:
+
+```json
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
+  "aliases": {
+    "components": "src/components",
+    "ui": "src/components/ui",
+    "utils": "src/lib/utils"
+  },
+  "style": "default",
+  "tailwind": {
+    "baseColor": "neutral",
+    "config": "./config/tailwind.config.js",
+    "css": "./src/index.css",
+    "cssVariables": true,
+    "prefix": "tw-",
+    "rsc": false,
+    "tsx": true
```

Comment:
Just noticed a minor typo error here in that the `rsc` and `tsx` parameters should be placed outside the `tailwind` section, instead of within.